src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -4057,7 +4057,7 @@ instruct rearrangeL(vReg dst, vReg src, vReg shuffle)
             n->bottom_type()->is_vect()->element_basic_type() == T_LONG);
   match(Set dst (VectorRearrange src shuffle));
   ins_cost(SVE_COST);
-  format %{ "sve_tbl $dst, D, $src, $shuffle\t# vector rearrange (D)" %}
+  format %{ "sve_tbl $dst, D, $src, $shuffle\t# vector rearrange (L)" %}
```

Comment:
I still think that it make no sense to separate these rules. But anyway, I can do a cleanup later. FYI: Xiaohong and our Intel friends are working hard trying not to increase the instruction selection patterns on masking support design: http://openjdk.java.net/jeps/8261663.